imperial.js

```diff
@@ -246,6 +255,21 @@ export default class Imperial {
     }
   }
 
+  handleTaxation(action) {
+    if (action.payload.slot === "taxation") {
+      const nationName = action.payload.nation;
+      const nation = this.nations.get(nationName);
+      const taxes =
+        this.factoryCount(nationName) * 2 +
+        this.flagCount(nationName) -
+        this.unitCount(nationName);
+      nation.treasury += taxes;
+      nation.taxChartPosition = "6";
```

Comment:
🧐

Reply:
I've been caught! What do you think? Pause here and actually figure out how to do this right or let the TDD make us fix it when we need to? (I do hope to resume and finish the 𝔖𝔠𝔥𝔫𝔢𝔩𝔩𝔢𝔦𝔫𝔰𝔱𝔢𝔦𝔤 someday)